sys-kernel/pf-sources/pf-sources-6.18_p2.ebuild

```diff
@@ -70,11 +70,17 @@ src_prepare() {
 	# When genpatches basic version is bumped, it also includes vanilla linux updates. Those are
 	# already in the -pf patch set, so need to remove the vanilla linux patches to avoid conflicts.
 	if [[ ${K_GENPATCHES_VER} -ne 1 ]]; then
+		find "${WORKDIR}"/ -type f -name '1710_disable_sse4a.patch' -delete ||
+			die "Failed to delete vanilla linux patches in src_prepare."
```

Comment:
Why delete this, is it already included in the pf patch set, or is it causing problems? Update the `die` message accordingly.